pypose/sparse/sparse_block_tensor.py

```diff
@@ -225,44 +244,45 @@ def coo_2_hybrid(coo, proxy, dense_proxy_thres: int=DENSE_SAFE_PROXY_THRES):
     proxy = proxy.coalesce()
 
     # Figure out the shape of the target Hybrid tensor.
-    s_dim = list(proxy.shape)
-    assert all(coo.shape[i] % s_dim[i] == 0 for i in range(proxy.dim())), \
-        f'coo and s_dim are not compatible: coo.shape = {coo.shape}, s_dim = {s_dim}. '
-    b_dim: List[int] = [coo.shape[i] // s_dim[i] for i in range(proxy.dim())] # block dimension.
+    shape_p = list(proxy.shape)
+    assert all(coo.shape[i] % shape_p[i] == 0 for i in range(proxy.dim())), \
+        f'coo and proxy shape are not compatible: coo.shape={coo.shape}, shape_p={shape_p}. '
+    shape_b: List[int] = [coo.shape[i] // shape_p[i] for i in range(proxy.dim())]
 
     # Create a temporary Hybrid tensor to represent the block sequence.
     coo = coo.coalesce()
     # in-block indices.
-    b_dim_t = torch.tensor(b_dim, dtype=torch.int64, device=coo.device).unsqueeze(-1)
-    in_block_indices = coo.indices() % b_dim_t
+    shape_b_t = torch.tensor(shape_b, dtype=torch.int64, device=coo.device).unsqueeze(-1)
+    inblock_indices = coo.indices() % shape_b_t
     # block indices.
-    block_indices = coo.indices() // b_dim_t
-    numel_proxy = torch.numel(proxy)
-    if numel_proxy < dense_proxy_thres:  # check the *dense* shape
+    indices_b = coo.indices() // shape_b_t
+    numel_p = torch.numel(proxy)
```

Comment:
Can put an additional comment here saying `torch.numel()` will return the dense number of element, not the sparse number.